client/src/components/login-signup/login.component.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from "react";
```

Comment:
We should follow standard naming convention for react components, i.e. `Login.js`